Dockerfile.server

```diff
@@ -1,50 +1,43 @@
-FROM node:16-bullseye
+### Build stage
+FROM node:16-bullseye AS builder
 
 RUN apt-get update \
     && apt-get install -y \
     python \
     && rm -rf /var/lib/apt/lists/*
 
+# Download the init tool Tini and make it executable for use in the final image
+ADD https://github.com/krallin/tini/releases/download/v0.19.0/tini-static /tini
+RUN chmod u+x /tini
+
 # Enables Yarn
 RUN corepack enable
 
-RUN echo "Node: $(node --version)" \
-    && echo "Npm: $(npm --version)" \
-    && echo "Yarn: $(yarn --version)"
-
-ARG user=joplin
-
-RUN useradd --create-home --shell /bin/bash $user
-USER $user
-
-ENV NODE_ENV production
-ENV RUNNING_IN_DOCKER 1
-EXPOSE ${APP_PORT}
-
-WORKDIR /home/$user
+RUN echo "Node: $(node --version)"
+RUN echo "Npm: $(npm --version)"
+RUN echo "Yarn: $(yarn --version)"
```

Comment:
@laurent22 The old-school Dockerfile best practice of combining multiple `RUN` commands into one does not apply when using multi-stage builds because the layers that are created during the build process don't end up in the final image. It's recommended to keep `RUN` commands separate during the build stage because it improves readability, maintainability and caching without influencing the size of the final image.